sw/device/tests/crypto/aes_gcm_functest.c

```diff
@@ -17,9 +17,11 @@ bool test_main(void) {
 
     // Call AES-GCM encrypt.
     uint32_t cycles = call_aes_gcm_encrypt(test);
+    (void)cycles;
 
     // Call AES-GCM decrypt.
     cycles = call_aes_gcm_decrypt(test, /*tag_valid=*/true);
+    (void)cycles;
```

Comment:
You can probably add this to the log if this is tracking execution cycles. Maybe just print the difference?

Reply:
Good point. Done.